docs/source/exposing.md

```diff
@@ -74,6 +74,36 @@ For the `LoadBalancer` type, Scylla Operator generates a LoadBalancer Service th
 On platforms with support for external load balancers, this Service provisions one. 
 The accessibility of this load balancer's address depends on the platform and any customizations made; in some cases it may be reachable from the internal network or public Internet.
 
+Customizations are usually managed via Service annotations, key-values provided in `annotations` field are merged into each Service object.
+For example, to expose LoadBalancer only to internal network use following annotations:
+
+EKS:
```

Comment:
![image](https://github.com/scylladb/scylla-operator/assets/32443348/d5521613-7f88-4fda-9e93-3edb4467d010)
Imo looks sort of ugly, maybe you could use bullets?
![image](https://github.com/scylladb/scylla-operator/assets/32443348/ef454141-7f70-4ea8-8d67-ebeee6c95acf)

Reply:
thanks, I added new extension which adds tabs:
![Selection_051](https://github.com/scylladb/scylla-operator/assets/3951334/64525a69-9009-490e-96c2-33701916a06f)

Reply:
Nice! Might be useful for other stuff as well, we could use the dropdowns for outputs or manifests.